lib/forest_liana/bootstraper.rb

```diff
@@ -100,6 +101,17 @@ def check_integrations_setup
       end
     end
 
+    def namespace_duplicated_models
+      _, models = ForestLiana.models
+        .group_by {|model| model.table_name}
+        .detect { |k, v| v.length > 1 }
```

Comment:
`detect` ruby method will detect only the first detection. It means that, if you have at least 2 couples of duplicated models, the feature won't work completely.